rb/lib/selenium/webdriver/common/interactions/pointer_actions.rb

```diff
@@ -18,7 +18,8 @@
 module Selenium
   module WebDriver
     module PointerActions
-      DEFAULT_MOVE_DURATION = 0.25 # 250 milliseconds
+      @default_move_duration = 0.25  # 250 milliseconds
```

Comment:
if you want to remove the constant, the best way of defining it will be as a private memoized method

```
private

def default_move_duration
  @default_move_duration ||= 0.25
end
```

Reply:
The reason for this PR is that the constant is being overridden in the Appium library, so this would change to specifically allow a writer.

Reply:
That's fair, I still wouldn't have procedurally defined instance vars, but each to their own.

Reply:
There's no reason to wrap the default in a method. Leverage the accessor.